fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -761,7 +761,7 @@ struct RawCursor {
 				int fragmentedWaste = kv.key.size() * (fragments - 1);
 
 				// Total bytes used for the fragmented case
-				int fragmentedTotal = kv.expectedSize() + fragmentedWaste;
+				//int fragmentedTotal = kv.expectedSize() + fragmentedWaste;
```

Comment:
This variable should not be removed, it is used by a debugging printf() call below which is commented out.

Reply:
Yeah, I think that was the reason it was commented out here rather than deleted. If you need the printf, then you can uncomment this line too. It may be better, though, to move this line down next to the printf for clarity.

Reply:
Oops, for some reason I was interpreting these edits as removals instead of deletions.